crates/red_knot_python_semantic/src/types.rs

```diff
@@ -3016,6 +3062,7 @@ impl<'db> KnownClass {
             "MethodWrapperType" => Self::MethodWrapperType,
             "WrapperDescriptorType" => Self::WrapperDescriptorType,
             "TypeAliasType" => Self::TypeAliasType,
+            "TypeVar" => Self::TypeVar,
```

Comment:
An unrelated bug that was exposed by the tests I added in this PR... turns out, we were never inferring `KnownClass::TypeVar` for any classes before now 😶